Dockerfile.rhel7

```
@@ -1,5 +1,5 @@
 
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.6 AS builder
```

Ah, yeah this needs to be `registry.ci...`.

```suggestion
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.6 AS builder
```